portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,78 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.io.*; 
+import java.lang.*; 
+import java.util.*; 
+import java.time.format.DateTimeFormatter;  
+import java.time.LocalDateTime;  
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comment data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  // Create comments Array
+  public String[][] comments = { {"Aidan Molloy", "Love the website", "12:00 - 01/01/2020"} };
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String userName = getParameter(request, "user-name", "");
+    String comment = getParameter(request, "comment", "");
+
+    // Get current date.
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("HH:mm - dd/MM/yyyy");
+    LocalDateTime now = LocalDateTime.now();
+
+    // Add the new comment to Array.
+    String[] newComment = {userName, comment, dtf.format(now)};
+    comments = addComment(comments.length, comments, newComment);
+
+    // Redirect user once comment added TODO: Redirect to Comments tab.
+    response.sendRedirect("/index.html#comments");
+  }
+
+  // Function to add a comment to the array
+  public static String[][] addComment(int arrayLength, String arr[][], String commentToBeAdded[])
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7-javadoc

Reply:
Updated documentation to follow style guide